modules/fedimint-mint-client/src/backup/recovery/tests.rs

```diff
@@ -1,717 +0,0 @@
-use std::collections::{BTreeMap, HashMap};
```

Comment:
**question:** Did this got deleted or moved somewhere?

Reply:
The tests got deleted. The differences between the old and new design where big and the new design seemed simple enough that the devimint integration tests for recovery seemed sufficient.